### ISSUE TITLE
Long IDs don't wrap nicely

### ISSUE THREAD

**Issue by ara4n** [matrix-org/matrix.to#188]
At least i think this is what https://twitter.com/RGBhackr/status/1347479932222509056 is complaining about

**Comment by bwindels** [comment]
still a problem

**Comment by niquewoodhouse** [comment]
Do we need to show those random character long IDs?  Maybe a generic term is more helpful to users/less alarming, like "Pick a way to view a the link below"?

**Comment by bwindels** [comment]
Yeah, could do. We'd probably need a different term for each type of link (room, user, ...) though?

**Comment by niquewoodhouse** [comment]
@bwindels we could start with something generic if it keeps things simpler, and just get feedback later.

**Comment by bwindels** [comment]
Can do, would probably be good to have some design for the "what do we show if all we have is a room id/user id/community id" case though.

**Comment by bwindels** [comment]
Doing this for now fwiw:
![image](https://user-images.githubusercontent.com/274386/114400751-cefa3780-9b91-11eb-842d-49e29ed8dc9d.png)
